### PR TITLE
👷 Migrate from jCenter to Maven Central

### PR DESCRIPTION
**Pull request by fitzsia2**
* Migrates from jCenter to Maven Central.
* Updates Gradle and Android Gradle Plugin.
* Changes `cz.ackee` packaging to `io.github.ackeecz`
* Changes artifact names
  * `core` -> `rx-social-login-core`
  * `google` -> `rx-social-login-google`
  * `facebook` -> `rx-social-login-facebook`

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: Publish
```
Comment (davidbilik) [inline .github/workflows/publish.yml:1]: Copy this from some existing project, it misses the slack notification steps

File: build.gradle [hunk 2]
```diff
@@ -5,12 +5,15 @@ buildscript {
     repositories {
         google()
         jcenter()
+        mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.3'
+        classpath 'com.android.tools.build:gradle:7.0.2'
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.3'
```
Comment (davidbilik) [inline build.gradle]: this should be removed along with 
```
classpath 'com.github.dcendents:android-maven-gradle-plugin:2.0'
```

File: build.gradle [hunk 3]
```diff
@@ -42,3 +46,5 @@ ext.libProperties.load(file("${rootDir}/lib.properties").newReader())
 ext {
     rxJavaVersion = '2.2.16'
```
Comment (davidbilik) [inline build.gradle]: is this the newest Rx version?

**Review by davidbilik — COMMENTED** [review]
I would add the maven central badge like we have in other libraries to facebook/google readmes

File: .github/workflows/publish.yml (hunk 1 above)
Comment (fitzsia2) [reply .github/workflows/publish.yml:1]: 👍  Added slack support in 977f286

File: build.gradle (hunk 2 above)
Comment (fitzsia2) [reply build.gradle]: Removed in 0d35727

File: build.gradle (hunk 3 above)
Comment (fitzsia2) [reply build.gradle]: Updated to 2.2.21 in e0f78fe

**Pull request merged**